lib/Fencer/Rules.hs

```diff
@@ -65,14 +65,28 @@ loadRulesFromDirectory
     let filteredFiles = if ignoreDotFiles
         then filter (not . isDotFile) files
         else files
-    (errs, rules) <- partitionEithers <$> mapM loadFile filteredFiles
-    pure $ if (null @[] errs) then Right rules else Left errs
+    (errs, mRules) <- partitionEithers <$> mapM loadFile filteredFiles
+    pure $ if (null @[] errs) then Right (catMaybes mRules) else Left errs
   where
-    loadFile :: FilePath -> IO (Either LoadRulesError DomainDefinition)
+    loadFile :: FilePath -> IO (Either LoadRulesError (Maybe DomainDefinition))
     loadFile file = catch
-      ((mapLeft (LoadRulesParseError file)) <$> Yaml.decodeFileEither @DomainDefinition file)
+      (parseErrorHandle file <$> Yaml.decodeFileEither @DomainDefinition file)
```

Comment:
This entire file clashes a lot with the current master, right?

Reply:
My apologies. There was a lot of shuffling due to numerous PR dependencies and then implementing all the feedback I got for the PRs.

Now the PR is up to date with master and you see only the actual changes. In particular, the line in `lib/Fencer/Rules.hs` you referred to here is non-existing now.

Reply:
> My apologies. There was a lot of shuffling due to numerous PR dependencies and then implementing all the feedback I got for the PRs.

That's perfectly understandable. I'll look into the updated PR in a minute.